config/initializers/assets.rb

```diff
@@ -9,3 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( hotjar.js )
```

Comment:
Change this to `Rails.application.config.assets.precompile += %w(hotjar.js)` to fix linting errors